ansible/inventory/teleop/group_vars/server.yml

```diff
@@ -7,3 +7,5 @@ nvidia_docker: false # Enable nvidia docker
 cyberglove: "A-penny" # Specify the branch of sr_cyberglove_config for cyberglove configuration
 use_aws: false # Flag to prompt for customer key for uploading files to AWS
 use_openvpn: false # Specify if openvpn-server should be installed
+setup_directory: "/usr/local/bin/"
+setup_script: "teleop-server-setup.sh"
```

Comment:
setup script should be hardcoded in task not as param

Reply:
Fixed in the last commit